templates/unstable-vite-cloudflare/.cloudflare.ts

```diff
@@ -0,0 +1,7 @@
+import { getBindingsProxy } from "wrangler";
+
+export const adapter = () => async () => {
```

Comment:
TODO: this file should be removed and this adapter should come from `@remix-run/cloudflare`

Reply:
actually from `@remix-run/dev` since we don't want to make `@remix-run/cloudflare` a build-time dependency